specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/examples/WafListAllPolicies.json

```
@@ -0,0 +1,102 @@
+{
+  "parameters": {
+    "api-version": "2020-04-01",
```

Here it should be `2020-05-01` since this is an example for WAF 2020-05-01
```suggestion
    "api-version": "2020-05-01",
```